### ISSUE TITLE
"Time Cursed" stat does not count time spent outside the overworld in Multiplayer

### ISSUE THREAD

**Issue by Umber0010** [Extegral/Enigmatic-Legacy#323]
Versions (Be specific, do not write "latest"):
 * Enigmatic Legacy: 2.21.1 (pack and Standalone)
 * Curios API: 5.0.7.0 (pack and Standalone)
 * Patchouli: 1.18.2-67 (Pack and Standalone)
 * Forge: 40.1.19 (pack) /40.1.0 (standalone)

Description of a Bug:
Certain items, such as the Infinitum, require that the player spends 99.5% of your "Time in the world spent cursed". However, if you're playing on a server, spending time outside of the overworld will result in the game not recognizing this as "time cursed".

FULL DISCLOSURE: This was discovered on the public server for TNP limitless 5. And while I tested it in singleplayer to confirm that this happened, I lack the means and knowledge of how I could test a multiplayer-only bug in a vacuum. 

#### How to Reproduce:
1. Join server
2. Equip ring of 7 curses
3. Spawn the Infinitum and watch "Time cursed" increase
4. Make nether portal
5. Enter nether
6. Watch "Time cursed" decrease

**Comment by StygianNights** [comment]
Properly Tracks in Singleplayer and LAN. I used a server I still had from Creeperhost, loaded the correct mods on plus JEI for simplicity, and my time tracker only ticks up in all dimensions I traveled to (Nether and End included). 
I wonder if maybe another mod is conflicting, and if you could provide a list of the mods you have to test for potential incompatibles?

**Comment by Umber0010** [comment]
Oh. Of course! Sorry for not being able to do myself, but the mod list is easy enough. Just the modpack TNP limitless 5
https://www.curseforge.com/minecraft/modpacks/tnp-limitless-5


V. 2.6.0 is the one you're looking for.

For full transparency, it's possible that it's something else causing the issue. Someone playing on the server with me at the time said that it was a dimension thing after I noticed that my time cursed stat dipped from 99.4% too 98.5%. 

Sorry I couldn't be more helpful here.

**Comment by StygianNights** [comment]
Okay, after a couple days of pulling my hair out, I got my Creeperhost server working so I can test this bug. Long story short, creeperhost was installing it wrong. 

Okay so, I cannot see the same issue you are describing at all. When I jump into nether or the end, neither one causes the issue of the time-worn tag decreasing when in either dimension. However, I have noticed that there is an issue of the time-worn tag not calculating properly during a player death. This is known and is being worked on. 

As an explanation of what might be happening: 
Considering that time without the ring is technically counted as 200 times more than when worn, this would make sense as to why you are having issues if you are dying a lot. For instance, during my test, over the course of 15 seconds at 78%, I increased by .5%, but when dying, lost a full .3% in the time it took me to respawn. So don't idle on the death screen for now.
